app/views/policy/document-downloads/pci_dss_attestation_of_compliance.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "../../layout.njk" %}
+
+{% block pageTitle %}
+  Policy Document Download - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">
+      GOV.UK Pay’s Payment Card Industry compliance
+    </h1>
+  </div>
+
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      GOV.UK Pay is certified as a level 1 service provider with the Payment Card Industry Data Security Standard (PCI DSS) version 3.2.1. The PCI DSS provides guidance to help maintain payment security. You can read more on our
+      <a href="https://www.payments.service.gov.uk/security/">security and compliance page</a>.
```

Comment:
I'm not sure if this somehow gets applied elsewhere but this won't fit the page format if it isn't given `class="govuk-link"`

Reply:
ah good spot, thanks I'll updated